apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -7,6 +7,7 @@ import Button from '@cdo/apps/templates/Button';
 import moduleStyles from './sections-refresh.module.scss';
 import {queryParams} from '@cdo/apps/code-studio/utils';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import PropTypes from 'prop-types';
```

Comment:
tiny nit: do you mind moving this up to line 2? We're not good at ordering imports here but generally, the React-related imports are first in the list. Don't block this PR on this though!